quasar.conf.js

```diff
@@ -14,7 +14,7 @@ module.exports = function(ctx) {
     supportIE: false,
     build: {
       scopeHoisting: true,
-      vueRouterMode: 'hash',
+      vueRouterMode: 'history',
```

Comment:
history need to use history mode ? why ?
already tested History mode with all the pages/features ?